int64 DroneID
string hostel_to
---
bool success
